packages/cadl-typescript/src/transform/transfromRLCOptions.ts

```diff
@@ -22,11 +21,6 @@ export function transformRLCOptions(
   const options = extractRLCOptions(program, emitterOptions);
   const batch = listClients(program);
   options.batch = batch;
-
-  // Fulfill the output dir if enabling sdk-folder in config
-  if (options["sdk-folder"] && isAbsolute(options["sdk-folder"])) {
```

Comment:
I think this logic would also be impacted if we don't replay on sdk-folder.

https://github.com/Azure/autorest.typescript/blob/75134f8e7aca5474ad60616196f6f7a32e06d611/packages/cadl-typescript/src/transform/transfromRLCOptions.ts#L176-L188